conmon-rs/common/proto/conmon.capnp

```diff
@@ -23,6 +23,7 @@ interface Conmon {
         exitPaths @3 :List(Text);
         oomExitPaths @4 :List(Text);
         logDrivers @5 :List(LogDriver);
+        cleanupCmd @6 :Text;
```

Comment:
Since we split it by whitespace, would it make sense to have this a `List(Text)` instead?